### DISCUSSION TITLE
Integrare Alexa in Home Assistant

### DISCUSSION THREAD

**Discussion by michele-diurno** [indomus/forum#3151] [HUB: Home Assistant]
Buongiorno a tutti,
sto rischiando la mia integrità mentale vi prego di aiutarmi.
Ho seguito la guida per integrare Alexa in HA. Purtroppo quando apro l'app Alexa su mio iphone non riesco a trovare la skill "haaska".
Il test di lambda è positivo ma mi sono arenato nel cercare haaska su Alexa.
Su Alexa developer console nello "Status" della skill haaska risulta "In dev". Devo fare una scelta dove dice "Actions"?
Nell'account linking vedo "You will need to configure the account linking skill capability in order for the Smart Home Voice Model to be functional".
Potreste darmi una dritta. 
Grazie mille.
Michele

**Comment by KC-inDomus** [comment]
devi ribattere tutto, punto per punto.

**Comment by michele-diurno** [comment]
Grazie per la pronta risposta.
Ci riprovo ma come si cancellano quelle voci relative al ruolo?
![immagine AWS](https://github.com/user-attachments/assets/92e63201-028b-44b9-a7d5-149658deee53)
Volevo chiedere anche quando mi registro  ad AWS posso utilizzare nome utente e psw uguali ad Amazon.com/.it ? Non credo che le devo cambiare in fin dei conti è una registrazione ex novo. Corretto?
Grazie mille e scusate se sono ottuso ma sono un novizio.

Aggiungo questa immagine perchè nel frattempo ho ripetuto tutto ma non riesco a trovare la skill "haaska"
![immagine account linking](https://github.com/user-attachments/assets/c53d904c-f439-4afe-a699-6883fb4fc953)
In settings ci sono due cursori potrebbero essere quelli a dare problemi?

**Comment by michele-diurno** [comment]
Ok non ci posso credere quanto possa essere vitale leggere attentamente la guida.
Non trovavo la scritta "le tue skill" perchè non scrollavo fino in fondo sul telefono.
Una volta fatto click su "le tue skill" sono riuscito a visualizzare la skill "haaska".
Grazie grazie per non avermi fatto arrendere!!!!

**Reply by KC-inDomus** [reply]
purtroppo è così - ci abbiamo sbattuto il muso tutti :)
